src/app/mi-details/view-story/view-story.page.ts

```diff
@@ -0,0 +1,147 @@
+import { Component, OnInit } from '@angular/core';
+import { finalize } from 'rxjs';
+import { LoaderService } from 'src/app/services/loader/loader.service';
+import { ProjectsApiService } from 'src/app/services/projects-api/projects-api.service';
+import { ToastService } from 'src/app/services/toast/toast.service';
+import  urlConfig  from 'src/app/config/url.config.json';
+import { UtilService } from '../../services/util/util.service';
+import { Clipboard } from '@capacitor/clipboard';
+import { Share } from '@capacitor/share';
+import { ShareLinkComponent } from '../../shared/share-link/share-link.component';
+import { PopoverController } from '@ionic/angular';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-view-story',
+  templateUrl: './view-story.page.html',
+  styleUrls: ['./view-story.page.scss'],
+})
+export class ViewStoryPage implements OnInit {
+  headerConfig:any={
+    showBackButton:true
+  }
+  projectId:any;
+  resource:any={ images: [], videos: [], documents: []};
```

Comment:
instead of using any use an interface when you're the resources to have this particular structure anyway